scripts/Compat.zs

```diff
@@ -30,4 +30,8 @@ recipes.addShapeless(<guideapi:ItemGuideBook:1>, [<minecraft:book>, <ore:crafter
 recipes.addShaped(<minecraft:torch> * 8, [[<ore:fuelCoke>], [<ore:stickWood>]]);
 
 // Diamond nuggets
-recipes.addShaped(<Translocator:diamondNugget>, [[<enhancedportals:nuggetDiamond>]]);
\ No newline at end of file
+recipes.addShaped(<Translocator:diamondNugget>, [[<enhancedportals:nuggetDiamond>]]);
```

Comment:
I suggest making the Enhanced Portal's Diamond Nugget uncraftable and use the Translocator nugget in their recipes. Unless they use Oredict already.